packages/gluestick/src/config/webpack/webpack.config.client.js

```diff
@@ -50,14 +52,35 @@ module.exports = (
     };
   });
   config.plugins.push(
-    new chunksPlugin(
-      deepClone(configuration),
-      { silent: settings.silent, chunk_info_filename: settings.chunk_info_filename },
-    ),
     // Make it so *.server.js files return null in client
     new webpack.NormalModuleReplacementPlugin(/\.server(\.js)?$/, path.join(__dirname, './mocks/serverFileMock.js')),
     progressHandler(logger, 'client'),
     new DuplicatePackageChecker(),
   );
+
+  // If vendor Dll bundle exists, use it otherwise fallback to CommonsChunkPlugin.
+  const vendorDllManifestPath: string = path.join(
+    process.cwd(), gluestickConfig.buildDllPath, manifestFilename,
+  );
+  if (fs.existsSync(vendorDllManifestPath)) {
+    logger.info('Vendor DLL bundle found, using DllReferencePlugin');
+    config.plugins.unshift(
+      new webpack.DllReferencePlugin({
+        context: configuration.context,
+        manifest: require(vendorDllManifestPath),
+      }),
+    );
+    config.plugins.push(new ChunksPlugin(deepClone(configuration), { appendChunkInfo: true }));
+  } else {
+    logger.info('Vendor DLL bundle not found, using CommonsChunkPlugin');
+    config.plugins.push(
+      new webpack.optimize.CommonsChunkPlugin({
+        name: 'vendor',
+        filename: `vendor${process.env.NODE_ENV === 'production' ? '-[hash]' : ''}.bundle.js`,
+      }),
```

Comment:
I didn't added it, I just moved it, I was like this from the version 1.0, so if it was working before, why to change it?